parse/table_definitions_arbitrum_nova/sushiswap/UniswapV2Pair_event_Approval.json

```diff
@@ -0,0 +1,54 @@
+{
+  "parser": {
+    "type": "log",
+    "contract_address": "SELECT pair FROM ref('UniswapV2Factory_event_PairCreated')",
```

Comment:
Ull need to create a PR with UniswapV2Factory_event_PairCreated first and just merge that 

or the dependency wont exist and the dag will fail